Gemfile.lock

```diff
@@ -408,8 +408,5 @@ DEPENDENCIES
   uglifier (~> 2.7.0)
   webmock
 
-RUBY VERSION
```

Comment:
Since there are no changes to the gems used, it is best to leave `Gemfile.lock` untouched.